src/Orc.DataAccess.Xaml/Controls/ConnectionStringBuilder/Views/ConnectionStringBuilder.cs

```diff
@@ -148,6 +165,27 @@ public override void OnApplyTemplate()
         {
             throw Log.ErrorAndCreateException<InvalidOperationException>("Can't find template part 'PART_ConnectionStringTextBox'");
         }
+
+        _connectionStringTextBox.TextChanged += OnTextChanged;
+    }
+
+    private void OnTextChanged(object sender, TextChangedEventArgs e)
+    {
+        if (_connectionStringTextBox is null)
```

Comment:
why this null check? Can't we use sender instead?

Reply:
sender is type of object... and still need to be checked by type